src/nidcpower/custom_types/lcr_load_compensation_spot.py

```diff
@@ -36,79 +36,66 @@ def __init__(self, data):
 class LCRLoadCompensationSpot(object):
     """Specifies a DUT specification for a given frequency to use in LCR load compensation."""
 
-    def __init__(
-        self,
-        frequency,
-        impedance=None,
-        ideal_capacitance=None,
-        ideal_inductance=None,
-        ideal_resistance=None,
-    ):
+    _lcr_reference_value_type_to_label_and_units = {
```

Comment:
If we're adding units to string representation of these, then we should do the same for `frequency` attribute of this struct and also for all the attributes of `LCRMeasurement` struct.

Reply:
Tracked by #1782